docs/Table.md

```diff
@@ -72,86 +99,215 @@ const { query } = initialize({ iotsitewiseClient });
+          resolution: 0
+        }
+      },
+      wind_speed: {
+        $cellRef: {
+          id: toId({ assetId: 'turbine-2', propertyId: 'wind_speed' }),
+          resolution: 0
+        }
+      }
+    }
+  ]}
+  
+  columnDefinitions={[
+    {
+      key: 'rpm',
+      header: 'RPM'
+    },
+    {
+      key: 'wind_speed',
+      header: 'Wind Speed'
     }
   ]}
   
   queries={[
     query.timeSeriesData({
       assets: [
         { 
-          assetId: 'engine-1', 
+          assetId: 'turbine-1', 
           properties: [
-            { propertyId: 'tempature' },
-            { propertyId: 'rpm' }
+            { propertyId: 'rotation_per_minute' },
+            { propertyId: 'wind_speed' }
           ],
-        }, { 
-          assetId: 'engine-2', 
+        }, 
+        { 
+          assetId: 'turbine-2', 
           properties: [
-            { propertyId: 'tempature' },
-            { propertyId: 'rpm' }
-          ],
+            { propertyId: 'rotation_per_minute' },
+            { propertyId: 'wind_speed' }
+          ]
         }
-    ])
-  ]}}
+      ]
+    })
+  ]}
 />
 ```
 
 ## Properties
 
-The table component contains the following properties that you can customize. 
+The table component contains the following customizable properties:
 
-### `tableColumns`
+Note: Since the table component built over the `@awsui/components-react` table component, please refer to [AWS-UI/table](https://github.com/aws/awsui-documentation/blob/main/components/table.md) about most of the available properties. Differences between AWSUI/Table and this Table component are listed below.
 
-Specifies the row and column structure for the table, provides a mechanism to map the time series data queried for, into the cells of your choice
-Type: Array
+### `items`
+Specifies the data displayed in rows. Each item contains the data for one row. The display of a row is handled by the cell property of each column definition in the columnDefinitions property.
 
-Each table column contains the following properties
+Type: Array of [Item](#item) object
 
-`header`
+Required: Yes
+#### `item`
+An item maps its own properties to dataStreams. A special property name [$cellRef](#cellref) can be used to refer to a datastream.
+```typescript jsx
+type Item = {
+  [key:string] : {
+    $cellRef: {
+      id: string,
+      resolution: number,
+    }
+  } | unknown
+}
 
-The text to be displays as the column header, the first row of the table
+const item: Item = {
+  rpm: {
+    $cellRef: {
+      id: 'dataStream-1',
+      resolution: 0
+    }
+  },
+  wind_speed: {
+    $cellRef: {
+      id: 'dataStream-2',
+      resolution: 0
+    }
+  },
+}
+```
+#### `$cellRef`
+$cellRef is a special keyword in Item. We use it to find the according dataStream. For the above example, 
+the item has two properties. Both of them are using `$cellRef` to map properties to data stream.
 
-Type: string
+Type: Object
```

Comment:
$cellRef doesn't have an explicit type?

Reply:
No it doesn't. But we have a type for Item which includes it.